mmdet3d/models/backbones/dgcnn.py

```diff
@@ -47,10 +52,10 @@ def __init__(self,
 
         self.GF_modules = nn.ModuleList()
         gf_in_channel = in_channels * 2
-        skip_channel_list = [gf_in_channel]  # input channel list
+        skip_channel_list: List[int] = [gf_in_channel]  # input channel list
```

Comment:
This seems no need to add typehint. And please also check other places.

Reply:
Thank you for your suggestion, I will fix it right now~